database/migrations/2014_10_12_000000_create_users_table.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateUsersTable extends Migration
+{
+    public function up()
+    {
+        Schema::create('users', function (Blueprint $table) {
+            $table->id();
+            $table->string('name');
+            $table->string('name_last');
+            $table->string('email')->unique();
+            $table->timestamp('email_verified_at')->nullable();
+            $table->string('password');
+            $table->rememberToken();
+            $table->text('bio')->nullable(); // Will be written in Markdown. The user profile image will come from Gravatar account for the email address.
+            $table->text('title')->nullable();
+            $table->softDeletes();
+            $table->timestamps();
```

Comment:
I think it will be best to change this initial migration back to the Laravel default for users when you run auth. Then add a separate migration with our modifications to it. I'll look into this before merging the PR.

Reply:
Yes, this is the best way to go. The user migration is included with all Laravel installations now, so we need to include it and the other 2 in this package. I updated our migration to just edit the existing user table.